### DISCUSSION TITLE
Docs: German & Polish Translations

### DISCUSSION THREAD

**Discussion by core-hacked** [saadeghi/daisyui#2681] [Show and tell]
# Creation of German and Polish translations
Hello, I have layed the groundwork for a German translation and will continue to pretty it up until it is ready. A polish translation will follow.
Progress can be followed on: https://github.com/core-hacked/daisyui/tree/master/src/docs/src/translation

Kind Regards!

**Comment by saadeghi** [comment]
Nice. Thanks!

**Comment by Avaray** [comment]
@core-hacked I've just submitted a PR [#2691](https://github.com/saadeghi/daisyui/pull/2691) with the Polish translation.

**Reply by core-hacked** [reply]
Lovely! Thanks
